test/Artifacts.toml

```diff
@@ -0,0 +1 @@
+../Artifacts.toml
```

Comment:
Not sure about this one ... is it recommended in the Julia doc? If not I would prefer to copy/paste the Artifacts.toml in the test folder

Reply:
Why are links bad?

Reply:
Okay, fixed in a clean way.